hrc/hrc/proto.hrc

```diff
@@ -125,6 +125,10 @@
     <location link="base/csharp.hrc"/>
     <filename>/\.cs$/i</filename>
   </prototype>
+  <prototype name="fsharp" group="main" description="F#">
```

Comment:
Could you please give a clear answer? Were you interested in my opinion or telling me to move it to `rare` in order to get PR accepted?

Thank you for your cooperation.